incubator/virtualcluster/config/setup/all_in_one.yaml

```diff
@@ -205,6 +218,7 @@ spec:
       creationTimestamp: null
       labels:
         app: vc-manager
+        virtualcluster-webhook: "true"
```

Comment:
Why do we need to add this label?

Reply:
the webhook service uses this label to select the vc-manager pod.